pkg/tsdb/models.go

```diff
@@ -85,3 +88,88 @@ func NewTimeSeries(name string, points TimeSeriesPoints) *TimeSeries {
 		Points: points,
 	}
 }
+
+type DataFrames interface {
```

Comment:
Would have been nice to have this in SDK/data package and use it in `backend.QueryDataResponse` instead of only encoded arrow bytes. The encoded arrow bytes makes sense sending over the gRPC wire so we want to keep that. But in Grafana we sometime want to just send the encoded arrow bytes straight to the frontend, but in other cases, e.g. alerting, we need the the decoded frames. 

Maybe too late to change that in SDK since it's a breaking change, why this PR suggests adding this in Grafana. Thoughts?